index.css

```diff
@@ -0,0 +1,752 @@
+  background-position: top 50px right 0%;
+}
+
+.contact-container {
+  padding: 60px 16px 45px;
+  color: #fff;
+}
+
+form {
+  padding: 0 22%;
+}
+
+form input,
+textarea {
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  padding: 15px;
+  min-height: 20px;
+}
+
+.contact-btn {
+  margin-top: 27px;
+}
+
+#message {
+  color: #fff;
+  font-size: 20px;
+  font-weight: 400;
+}
+
+#contact-me {
+  font-size: 40px;
+  font-weight: 700;
+  margin-bottom: 1rem;
+}
+
+.no-pad {
+  padding-inline-start: 0;
+}
+
+/* flex */
+.d-flex {
+  display: flex;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.justify-start {
+  justify-content: flex-start;
+}
+
+.align-center {
+  align-content: center;
+}
+
+.align-item-center {
+  align-items: center;
+}
+
+.align-self-center {
+  align-self: center;
+}
+
+.flex-nowrap {
+  flex-wrap: nowrap;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.space-y-3 > * + * {
+  margin-top: 8%;
+}
+
+.space-y-1 > * + * {
+  margin-top: 1rem;
+}
+
+.space-x-1 > * + * {
+  margin-left: 8px;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.btn-wrappper {
+  text-align: center;
+}
+
+.font-40 {
+  font-size: 40px;
+  font-weight: 700;
+  margin-bottom: 20px;
+}
+
+.font-17 {
+  font-size: 1.05rem;
+}
+
+/* margin */
+.m0 {
+  margin: 0;
+}
+
+.m5 {
+  margin: 5%;
+}
+
+.m10 {
+  margin: 10%;
+}
+
+.m15 {
+  margin: 15%;
+}
+
+.m20 {
+  margin: 20%;
+}
+
+.m25 {
+  margin: 25%;
+}
+
+.mb-48 {
+  margin-bottom: 28px;
+}
+
+.mt-100 {
+  margin-top: 96px;
+}
+
+.font-13 {
+  font-size: 18px;
+}
+
+.h-0 {
+  margin-block-start: 0;
+}
+
+.f-color {
+  color: #091e42;
+}
+
+/* Margin left right */
+.mx-8 {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.mx-12 {
+  margin-left: 12px;
+  margin-right: 12px;
+}
+
+.mx-18 {
+  margin-left: 18px;
+  margin-right: 18px;
+}
+
+.mx-28 {
+  margin-left: 28px;
+  margin-right: 28px;
+}
+
+/* Margin Top Bottom */
+.my-8 {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.my-12 {
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.my-18 {
+  margin-top: 18px;
+  margin-bottom: 18px;
+}
+
+.my-24 {
+  margin-top: 24px;
+  margin-bottom: 24px;
+}
+
+.my-28 {
+  margin-top: 28px;
+  margin-bottom: 28px;
+}
+
+.my-36 {
+  margin-top: 36px;
+  margin-bottom: 36px;
+}
+
+.my-88 {
+  margin-top: 8%;
+  margin-bottom: 8%;
+}
+
+.my-52 {
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.mb-32 {
+  margin-bottom: 32px;
+}
+
+.mt-40 {
+  margin-top: 40px;
+}
+
+/* padding */
+.p5 {
+  padding: 5%;
+}
+
+.p10 {
+  padding: 10%;
+}
+
+.p15 {
+  padding: 15%;
+}
+
+.p20 {
+  padding: 20%;
+}
+
+.p25 {
+  padding: 25%;
+}
+
+.pb-100 {
+  padding-bottom: 100px;
+}
+
+.wx-801 {
+  width: 801px;
+}
+
+.foward,
+.downward {
+  font-size: 23px;
+  color: #acb7c3;
+}
+
+.list-style-n {
+  list-style: none;
+}
+
+.msg-wrap {
+  padding: 0 15%;
+}
+
+/* media query */
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] As suggested by the previous reviewer, make use of `min-width: 768px` to style for desktop designs because mobile designs should come first as your portfolio should be mobile first. 
- [ ] Kindly make your style mobile first by not using `@media only screen and (max-width: 768px) {` to style for mobile, instead use a mobile first design with `@media only screen and (min-width: 768px) {` to style for desktops.